scripts/initscripts/snapd.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Starts and stops a single snapd instance on this system
+Documentation=https://github.com/intelsdi-x/snap
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+User=root
+Group=root
+Type=simple
+Restart=on-failure
+ExecStart=/usr/local/bin/snapd --config=/etc/snapd/snapd.yaml
```

Comment:
The default is the .conf file, any preference for using yaml? I can see if we can make it an option in the service script.